docs/linux-instructions.md

```diff
@@ -88,6 +88,10 @@ shortcuts.
  3. You can launch the application by executing `dotnet run` in the `src/WebUI` directory.
 
  4. You can run the tests by executing `dotnet test` in the root directory.
+ 
+ ## Setting up the auth provider
```

Comment:
```suggestion
 ## Setting Up The Authentication Provider
```

Reply:
I'll not change "up the" for consistency with the other headings.